### ISSUE TITLE
build command not reporting error with layouts

### ISSUE THREAD

**Issue by jobearrr** [twostraws/Ignite#64]
On my `Site` definition, I had the layouts assigned like below:
``` swift
var layouts = [
    BlogPost()
]
```
where `BlogPost` is a custom layout inheriting from `ContentPage`.

When I run the `ignite build` command, it finishes with the message `✅ Successfully built!`, but I look at the `Build/` folder and the contents using the `BlogPost` layout are not there.

I ran the executable scheme on Xcode and there I got an error `Failed to find layout named BlogPost.`

Should there be a difference between these two building options?
I think it would be nice to have the `build` command reporting that error which I got reported when running the executable scheme on Xcode.

**Comment by jobearrr** [comment]
Yes, the problem was solved by specifying the type.

**Comment by twostraws** [comment]
Thank you! I'm going to leave this issue open because I think we should document this on the Ignite Samples site, so this is a great first issue for someone.

**Comment by jkaunert** [comment]
Xcode autocomplete adds the type when you accept, FYI.

**Comment by twostraws** [comment]
Yes, but I don't think it's terribly clear to many people what the distinction is. I think we should definitely look to make this clear on IgniteSamples somewhere.

**Comment by JPToroDev** [comment]
Addressed in IgniteSamples in the latest commit.
